src/Components/TeamPage/TeamPage.js

```diff
@@ -1,128 +1,127 @@
 import React from "react";
 import Header from "../Marketing Page/Header";
-import { makeStyles, useTheme } from '@material-ui/core/styles';
-import { Grid } from '@material-ui/core';
-import { Link } from '@material-ui/core';
-
-import { data17, data19 } from "./TeamPageData.js"
+import { useSpring, animated } from "react-spring";
+import { makeStyles, useTheme } from "@material-ui/core/styles";
```

Comment:
On the netlify deployment it advises that it cannot find the module for react-spring

Reply:
I was able to add react-spring to the pull request somehow it hadn't been added to the pull request in a commit.